packages/fxa-react/package.json

```diff
@@ -3,13 +3,13 @@
   "version": "0.0.0",
   "description": "Shared components for FxA React Apps",
   "exports": {
-    "./components/": "./dist/packages/fxa-react/components/",
+    "./components/": "./components/",
     "./configs/tailwind": "./configs/tailwind.js",
     "./configs/rescripts": "./configs/rescripts.js",
     "./configs/storybooks": "./configs/storybooks.js",
     "./images/": "./images/",
     "./extract-imported-components": "./extract-imported-components.js",
-    "./lib/": "./dist/packages/fxa-react/lib/"
+    "./lib/": "./lib/"
```

Comment:
These changes effectively get Webpack 5 to do what Webpack 4 was doing: loading and transpiling the original source.